platform/linuxbsd/export/export_plugin.cpp

```diff
@@ -79,31 +79,21 @@ Error EditorExportPlatformLinuxBSD::export_project(const Ref<EditorExportPreset>
 	return err;
 }
 
-void EditorExportPlatformLinuxBSD::set_extension(const String &p_extension, const String &p_feature_key) {
-	extensions[p_feature_key] = p_extension;
-}
-
 String EditorExportPlatformLinuxBSD::get_template_file_name(const String &p_target, const String &p_arch) const {
 	return "linux_" + p_target + "." + p_arch;
 }
 
 List<String> EditorExportPlatformLinuxBSD::get_binary_extensions(const Ref<EditorExportPreset> &p_preset) const {
 	List<String> list;
-	for (const KeyValue<String, String> &E : extensions) {
-		if (p_preset->get(E.key)) {
-			list.push_back(extensions[E.key]);
-			return list;
-		}
-	}
-
-	if (extensions.has("default")) {
-		list.push_back(extensions["default"]);
-		return list;
-	}
-
+	list.push_back(p_preset->get("binary_format/architecture"));
 	return list;
 }
 
+void EditorExportPlatformLinuxBSD::get_export_options(List<ExportOption> *r_options) {
+	EditorExportPlatformPC::get_export_options(r_options);
+	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "binary_format/architecture", PROPERTY_HINT_ENUM, "x86_64,x86_32,arm64,arm32,rv64,ppc64,ppc32"), "x86_64"));
```

Comment:
I'm a bit concerned by the UX as we won't be providing export templates for several of those, and that will be confusing to users. What does it show currently if you select one of them?

Reply:
It's the same error as selecting x86_64 when no export templates are installed, it shows a red message that the file is missing ("No export template found at the selected path"). Users can either switch to a different arch, or go add some new files. The manage export templates page lets users open the export templates folder.

![export-arch-missing](https://user-images.githubusercontent.com/1646875/181842519-69ba0f14-9aae-4c9f-ac95-4ad80c5818c7.png)

Reply:
Alright, should be good enough for now. We'll need to improve this UX as it's quite user unfriendly for now IMO.